src/test-run/index.js

```diff
@@ -178,6 +179,9 @@ export default class TestRun extends Session {
         if (this.pendingRequest)
             this._resolvePendingRequest(command);
 
+        if (command.type === COMMAND_TYPE.wait)
```

Comment:
I'm agree with this, so we should execute the `wait` command before `pendingJsError` check